calculator.app/calculator.py

```diff
@@ -0,0 +1,18 @@
+def add(a, b):
+    """Addition feature"""
```

Comment:
Please add more details on every docstring. 

- What the function is do
- What the function will return 
- What is the details of the parameter? ie Should it be an interger or string?